config

```diff
@@ -1,3 +1,370 @@
 ngx_addon_name=ngx_http_lua_upstream_module
-HTTP_MODULES="$HTTP_MODULES ngx_http_lua_upstream_module"
-NGX_ADDON_SRCS="$NGX_ADDON_SRCS $ngx_addon_dir/src/ngx_http_lua_upstream_module.c"
+HTTP_LUA_UPSTREAM_SRCS="$ngx_addon_dir/src/ngx_http_lua_upstream_module.c"
+
+ngx_feature="Lua library"
+ngx_feature_libs="-llua -lm"
```

Comment:
This is too much. I don't think ngx_lua_upstream should link against Lua. It already depends on ngx_lua anyway.